server/routers/index.ts

```diff
@@ -1,10 +1,10 @@
 const express = require('express');
-// const searchController = require('./searchController');
+const searchController = require('./searchController');
 // const eventsController = require('./eventsController');
 // const profileController = require('./profileController');
 // const reviewsController = require('./reviewsController');
 const router = express.Router();
 
-
+router.route('/search').get(searchController.handleSearch);
```

Comment:
changing it to `router.route('search/:search).get(searchController.handleSearch)`